src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -9,12 +9,13 @@
 
 namespace System.Runtime.Intrinsics
 {
+    [StructLayout(LayoutKind.Sequential, Size = 8)]
+    public struct Vector64<T> where T : struct {}
```

Comment:
Are we including private fields in the reference assemblies now? I think I remember some thread on the topic...

Asking since we added private fields here: https://github.com/dotnet/coreclr/pull/15897

Reply:
Also not clear on if we need the `[Intrinsic]` attribute (which was already missing) or if we need the `Debugger*` attributes, which are new.

Reply:
> Are we including private fields in the reference assemblies now? I think I remember some thread on the topic...

Yes.  See the following for more details.

https://github.com/dotnet/buildtools/pull/1859
https://github.com/dotnet/corefx/pull/26286/
http://blog.paranoidcoding.com/2016/02/15/are-private-members-api-surface.html

Reply:
For the DebuggerTypeProxy attributes, I don't see any others in a `ref` file.  So I'd assume we don't need them in the ref assembly.  Plus, since it points to a type that is internal, it can't be.  And lastly, the debugger doesn't know anything about 'ref' assemblies.

The `[Intrinsic]` attribute is internal, right?  So it can't be in the ref assembly.